workspaces/bulk-import/packages/backend/package.json

```diff
@@ -37,15 +37,17 @@
     "@backstage/plugin-permission-node": "^0.10.3",
     "@backstage/plugin-proxy-backend": "^0.6.5",
     "@backstage/plugin-scaffolder-backend": "^2.2.0",
+    "@backstage/plugin-scaffolder-backend-module-github": "^0.8.2",
     "@backstage/plugin-search-backend": "^2.0.5",
     "@backstage/plugin-search-backend-module-catalog": "^0.3.7",
     "@backstage/plugin-search-backend-module-pg": "^0.5.47",
     "@backstage/plugin-search-backend-module-techdocs": "^0.4.5",
     "@backstage/plugin-search-backend-node": "^1.3.14",
     "@backstage/plugin-techdocs-backend": "^2.0.5",
     "@red-hat-developer-hub/backstage-plugin-bulk-import-backend": "workspace:^",
+    "@roadiehq/scaffolder-backend-module-utils": "^3.6.0",
```

Comment:
Do we need this?

I think its fine, but we need to be sure that we have also templates that doesn't require this, or?